transaction.go

```diff
@@ -182,8 +183,8 @@ func (t *Transaction) SetReferenceBlockID(blockID Identifier) *Transaction {
 	return t
 }
 
-// SetGasLimit sets the gas limit for this transaction.
-func (t *Transaction) SetGasLimit(limit uint64) *Transaction {
+// SetComputeLimit sets the gas limit for this transaction.
```

Comment:
```suggestion
// SetComputeLimit sets the compute limit for this transaction.
```

Reply:
https://github.com/onflow/flow-go-sdk/pull/448/commits/e78be7212693c76d49638d68194f603716152485

There is still "gas" in protobuf and other places and will involve changing other places as well.